src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -366,6 +372,68 @@ public static void closeProjectViewTree(RemoteRobot remoteRobot, String appName)
     }
 
+    public static void openOrCreateEnvCfgFile(RemoteRobot remoteRobot, String projectName, String fileName) {
+        // Click on File on the Menu bar.
+        ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofMinutes(2));
+
+        // hide the terminal window for now
+        UIBotTestUtils.hideTerminalWindow(remoteRobot);
+
+        // get a keyboard - should maybe make this class wide
+        Keyboard keyboard = new Keyboard(remoteRobot);
+
+        // get a JTreeFixture reference to the file project viewer entry
+        JTreeFixture projTree = projectFrame.getProjectViewJTree(projectName);
+        projTree.expand(projectName, "src", "main", "liberty", "config");
+        if (!projTree.hasText(fileName)) {
+            // use the JTree ProjectTree to one step expand the project tree
+            projTree.findText("config").rightClick();
+
+            // use the projectFrame to pick up the file create menu
+            projectFrame.getActionMenu("New").click();
+            projectFrame.getActionMenuItem("File").click();
+            keyboard.enterText(fileName);
+            keyboard.enter();
+            try {
+                remoteRobot.find(DialogFixture.class, byXpath("//div[@class='JCheckBox']"), Duration.ofSeconds(2)).click();
+                remoteRobot.find(DialogFixture.class, byXpath("//div[@text.key='button.cancel']"), Duration.ofSeconds(2)).click();
+            } catch (WaitForConditionTimeoutException e) {
+                // checkbox/cancel button has already been clicked, nothing to do
+            }
+        } else {
+            // if either of these files exist in the test app mark them for non-deletion
+            // when the test cases cleanup at the end
+            if (fileName.equals("server.env")) {
+                deleteServerEnv = false;
+            }
+            else if (fileName.equals("bootstrap.properties")){
+                deleteBootstrapProps = false;
+            }
+            projTree.findText(fileName).doubleClick();
+        }
+    }
+
+    public static void deleteEnvCfgFile(RemoteRobot remoteRobot, String projectName, String fileName) {
+
+        Keyboard keyboard =  new Keyboard(remoteRobot);
+        // Click on File on the Menu bar.
+        ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofMinutes(2));
+        // get a JTreeFixture reference to the file project viewer entry
+        JTreeFixture projTree = projectFrame.getProjectViewJTree(projectName);
+        projTree.expand(projectName, "src", "main", "liberty", "config");
+        if (!projTree.hasText(fileName)) {
+            // file is not present, nothing to delete
+        } else {
+            // delete either server.env or bootstrap.properties if they were
+            // not part of the original app ie: if the testcases created them.
+            if (deleteServerEnv || deleteBootstrapProps) {
```

Comment:
I think this logic could lead to a file being deleted that is not marked for deletion. For example, if the method is called with "bootstrap.properties" passed in, and `deleteServerEnv` is true but `deleteBootstrapProps` is false, it looks like "bootstrap.properties" would still get deleted.